src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Llm/EventHelper.cs

```diff
@@ -57,6 +58,20 @@ public static class EventHelper
 
             agent.RecordLlmEvent("LlmChatCompletionSummary", attributes);
 
+            if (isError)
```

Comment:
Interesting approach to getting the error attributes.  I was attempting to get them later on, but I think this works!